[Federation] Subscribers -> Followers shows empty handle

### Describe the bug
I just upgraded my WriteFreely deployment (pull latest from git, rebuild, restart service, migrate db), and I found the new "Subscribers" view. Looks great! However, it does not show any subscribers for me, but rather a single `@` character which links to my Mastodon account (I follow my own blog so I can comment on it, so I think it is successfully parsing only one follower).
I see the following in the logs:

```
ERROR: 2023/10/12 22:24:34 templates.go:58: Error parsing subscribers: template: subscribers.tmpl:49:36: executing "subscribers" at <.EstimatedHandle>: can't evaluate field EstimatedHandle in type writefreely.RemoteUser
2023/10/12 22:24:34 "GET /me/c/mat/subscribers?filter=fediverse" 200 2.07474ms "Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/118.0"
```
Screenshot:
![image](https://github.com/writefreely/writefreely/assets/44881120/c93ddca2-538b-4cc2-8196-6a5a8be9dfc8)


### Expected behavior
A list of all Fediverse followers should be shown.

### Application configuration
 - **Single mode or Multi-user mode?** Single user
 - **Database?** sqlite
 - **Open registration?** no
 - **Federation enabled?** yes

**Version or last commit**:
3870749e5e686af96910cd63320440274dddc595

--- a/activitypub.go
+++ b/activitypub.go
@@ -65,6 +65,20 @@ type RemoteUser struct {
 	SharedInbox string
 	URL         string
 	Handle      string
+	Created     time.Time
+}
+
+func (ru *RemoteUser) CreatedFriendly() string {
+	return ru.Created.Format("January 2, 2006")
+}
+
+func (ru *RemoteUser) EstimatedHandle() string {
+	if ru.Handle != "" {
+		return ru.Handle
+	}
+	username := filepath.Base(ru.ActorID)
+	host, _ := url.Parse(ru.ActorID)
+	return username + "@" + host.Host
 }
 
 func (ru *RemoteUser) AsPerson() *activitystreams.Person {

